demos/embeddings/barbell-graphwave.ipynb

```diff
@@ -0,0 +1,167 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "trans_emb = PCA(n_components=2).fit_transform(np.vstack(embeddings))\n",
+    "\n",
+    "plt.scatter(\n",
+    "    trans_emb[:, 0], trans_emb[:, 1], c=classes, cmap=\"jet\", alpha=0.7,\n",
+    ")\n",
+    "plt.show()"
```

Comment:
This plot still looks very different to the one in the paper. Did you get the parameters correct?

<img width="379" alt="image" src="https://user-images.githubusercontent.com/1203825/74780819-7feda500-52f4-11ea-8fa7-12f780b8d8a9.png">

<img width="349" alt="image" src="https://user-images.githubusercontent.com/1203825/74780809-76643d00-52f4-11ea-9645-adb6071b8061.png">